### PR TITLE
Add Server Plantaria.net

### PR DESCRIPTION
**Pull request by xXSwainXx**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by toohard2explain** [comment]
is the server available to the public?

**Comment by xXSwainXx** [comment]
the server is currently under maintenance

**Comment by toohard2explain** [comment]
all laby.net servers are active and public available servers

**Comment by xXSwainXx** [comment]
our old server was also added to maintenance back then? we just have maintenance on now because we are adding something new but will be online soon?! you can still show presence on laby.net

**Comment by xXSwainXx** [comment]
> all laby.net servers are active and public available servers

https://laby.net/server/neruxvace (offline)
https://laby.net/server/introuble (offline / maintenance)
https://laby.net/server/maybemc (maintenance)
playlegend was also on laby.net while the maintenance :/

**Comment by toohard2explain** [comment]
then open a new pull request to remove the listed servers

**Comment by toohard2explain** [comment]
> > all laby.net servers are active and public available servers
> 
> https://laby.net/server/neruxvace (offline) https://laby.net/server/introuble (offline / maintenance) playlegend was also on laby.net while the maintenance :/ https://laby.net/server/amuria (maintenance) https://laby.net/server/astrogames (offline) and many more i don't have time to look for other networks that are in maintenance or offline :)

some of them are also re-opening soon for the other see #880

**Comment by xXSwainXx** [comment]
> some of them are also re-opening soon for the other see #880

yeah the new name is "amuria" and not more "astrogames"... and please don't start discussing here it is still the work of the maintainer here to accept the server or not...

File: minecraft_servers/plantaria/manifest.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "%.plantaria.net",
+    "%.plantaria.dev"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://plantaria.net",
+    "web_shop": "https://store.plantaria.net",
```
Comment (DoJapHD) [inline minecraft_servers/plantaria/manifest.json:15]: Not reachable

File: minecraft_servers/plantaria/manifest.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "server_wildcards": [
+    "%.plantaria.net",
+    "%.plantaria.dev"
+  ],
+  "supported_languages": [
+     "de",
+     "en"
+  ],
+  "social": {
+    "web": "https://plantaria.net",
```
Comment (DoJapHD) [inline minecraft_servers/plantaria/manifest.json:14]: Remove website if non-existent yet

File: minecraft_servers/plantaria/manifest.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "web_shop": "https://store.plantaria.net",
+    "web_support": "https://plantaria.net/support",
```
Comment (DoJapHD) [inline minecraft_servers/plantaria/manifest.json:16]: Remove website if non-existent yet

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

File: minecraft_servers/plantaria/manifest.json (hunk 1 above)
Comment (xXSwainXx) [reply minecraft_servers/plantaria/manifest.json]: the website is now removed and will only be added when it is out of maintenance.

**Pull request merged**